src/Components/CreateImageWizard/steps/Locale/components/LanguagesDropDown.tsx

```diff
@@ -28,6 +28,24 @@ import sortfn from '../../../../../Utilities/sortfn';
 import { useLocaleValidation } from '../../../utilities/useValidation';
 import { languagesList } from '../languagesList';
 
+const parseLanguageOption = (language: string) => {
+  try {
+    const [region, encoding] = language.split('.');
+    const [languageCode, countryCode] = region.split('_');
+
+    const languageName = new Intl.DisplayNames(['en'], {
+      type: 'language',
+    }).of(languageCode);
+    const countryName = new Intl.DisplayNames(['en'], { type: 'region' }).of(
+      countryCode
+    );
+
+    return `${languageName} (${countryName} - ${encoding})`;
```

Comment:
```suggestion
    return `${languageName} - ${countryName} (${language})`;
```

I guess that's what my suggestion would look like.